### PR TITLE
feat: add `google.api.ResourceDescriptor.Style`, require python >=3.6

### LINKED ISSUE #23: Python 3.9: DeprecationWarning: Call to deprecated create function FileDescriptor()
#### Environment details

  - OS type and version: MacOS 11.0.1
  - Python version: 3.9.0
  - pip version: 20.2.4
  - `googleapis-common-protos` version: 1.52.0

#### Steps to reproduce

  1. `python -W default -c 'import google.rpc.status_pb2'`

#### Code example

N/A

#### Stack trace
```
/.../python3.9/site-packages/google/rpc/status_pb2.py:33: DeprecationWarning: Call to deprecated create function FileDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
  DESCRIPTOR = _descriptor.FileDescriptor(
/.../python3.9/site-packages/google/rpc/status_pb2.py:50: DeprecationWarning: Call to deprecated create function FieldDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
  _descriptor.FieldDescriptor(
/.../python3.9/site-packages/google/rpc/status_pb2.py:68: DeprecationWarning: Call to deprecated create function FieldDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
  _descriptor.FieldDescriptor(
/.../python3.9/site-packages/google/rpc/status_pb2.py:86: DeprecationWarning: Call to deprecated create function FieldDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
  _descriptor.FieldDescriptor(
/.../python3.9/site-packages/google/rpc/status_pb2.py:43: DeprecationWarning: Call to deprecated create function Descriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
  _STATUS = _descriptor.Descriptor(
```

### PR DESCRIPTION
**Pull request by busunkim96**
feat: add `google.api.ResourceDescriptor.Style`
fix: add create_key to FieldDescriptors
    Fixes #23 Regenerate the `_pb2.py` files with the latest version of protoc (3.14.0)

### CODE REVIEW

File: google/longrunning/operations_pb2_grpc.py [hunk 1]
```diff
@@ -1,19 +1,3 @@
-# -*- coding: utf-8 -*-
-
-# Copyright 2020 Google LLC
```
Comment (tswast) [inline google/longrunning/operations_pb2_grpc.py:3]: Is it OK for this license header to be removed?

**Review by tswast — COMMENTED** [review]
Do we need to update the minimum protobuf version in `setup.py`?

File: google/longrunning/operations_pb2_grpc.py (hunk 1 above)
Comment (busunkim96) [reply google/longrunning/operations_pb2_grpc.py:3]: Oh nope that shouldn't have been removed. Re-added.

**Review by parthea — APPROVED** [review]

**Comment by busunkim96** [comment]
> Do we need to update the minimum protobuf version in setup.py?

That's a good point, I'm not really sure. I'll add constraints files so the lower bound is tested.

**Pull request merged**